packages/web/src/js/Collapse.ts

```diff
@@ -42,9 +44,9 @@ class Collapse extends BaseComponent {
       hideOnCollapse: !!(this.element.dataset.more || this.element.dataset.more === ''),
```

Comment:
You can check for `data-parent` here in the constructor as you do it for other datasets.

Reply:
fixed